docs/user/openstack/README.md

```diff
@@ -106,68 +106,19 @@ openstack network list --long -c ID -c Name -c "Router Type"
 
 ## Isolated Development Environment
 
-If you would like to set up an isolated development environment, you may use a bare metal host running CentOS 7. The following repository includes some instructions and scripts to help with creating a single-node OpenStack development environment for running the installer. Please refer to [this documentation](ocp-doit) for further details.
-
-[ocp-doit]: https://github.com/shiftstack-dev-tools/ocp-doit
+If you would like to set up an isolated development environment, you may use a bare metal host running CentOS 7. The following repository includes some instructions and scripts to help with creating a single-node OpenStack development environment for running the installer. Please refer to [this documentation](https://github.com/shiftstack-dev-tools/ocp-doit) for further details.
 
 ## Initial Setup
 
-If you don't already have the OpenShift Client Tools, follow [these steps](https://docs.openshift.com/enterprise/3.0/cli_reference/get_started_cli.html). Once you have that set up, download the latest stable versions of the [installer](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/). You will need to unzip the tarball with the following command:
+Download the [latest versions](https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest/) of the OpenShift Client and installer and uncompress the tarballs with the `tar` command:
 
 ```sh
 tar -xvf openshift-install-OS-VERSION.tar.gz
 ```
 
-For the time being, the OpenStack platform is not supported in the prompts. We will instead provide you with a template `install-config.yaml` to use below. If you want to go beyond the basic customizations we display below, there is more information available in [this document](customization.md). We have labled sections in the yaml below with the following syntax: (#). Look for the numbers below the template for explanations of how to complete those sections.
+You could either use the interactive wizard to configure the installation or provide an `install-config.yaml` file for it. See [all possible customization](customization.md) via the `install-config.yaml` file.
 
-```yaml
```

Comment:
I kinda liked having this, I feel like it was a good way to communicate how to use the install config

Reply:
Why dont we move this to customization.yaml?

Reply:
It was duplicating the info from the customization.md doc, so it had to move.

Now, idk if we want to follow the same format in customization.md... I also like how customization.md is consistent across the different platforms.

Reply:
Yeah, I suppose we should keep it consistent with what other platforms are doing